docs/pages/guides/spend-permissions/api-reference/wallet-fetchpermissions.mdx

```diff
@@ -0,0 +1,43 @@
+## Coinbase Wallet API
+
+### `wallet_fetchPermissions`
+
```

Comment:
Perhaps a one liner here about what this solves to help frame it in the wider context

Reply:
i.e. a utility API endpoint for recalling permissions previously granted for a specific user